### PR TITLE
Add unit tests for the `context.py` module

### PR DESCRIPTION
**Pull request by mihaelabalutoiu**
This PR adds unit tests for the `context.py` module.

### CODE REVIEW

File: coriolis/tests/test_context.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        self.assertEqual(new_context.roles, [])
+
+    def test_to_dict(self):
+        result = self.req_context.to_dict()
+
+        self.assertEqual(result['timestamp'],
+                         self.req_context.timestamp.isoformat())
+
+    def test_from_dict(self):
+        result = self.req_context.to_dict()
```
Comment (Dany9966) [inline coriolis/tests/test_context.py:53]: Here you don't need to check whether the timestamp is handled, you already do that in your `__init__` test. I think you should either:
1. Check that the class is called with `**result`
2. Remove the test case altogether. It's fine if `from_dict` is not covered, it doesn't really do much.

**Review by Dany9966 — CHANGES_REQUESTED** [review]
Mostly LGTM

File: coriolis/tests/test_context.py (hunk 1 above)
Comment (mihaelabalutoiu) [reply coriolis/tests/test_context.py]: Gotcha :) Thank you!

**Comment by mihaelabalutoiu** [comment]
@Dany9966 I updated the code, please take a look now, thank you!

File: coriolis/tests/test_context.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    def test_can_policy_not_authorized(self, mock_check_policy):
+        mock_check_policy.side_effect = exception.PolicyNotAuthorized(
+            action='test_action')
+
+        self.assertRaises(exception.PolicyNotAuthorized,
+                          self.req_context.can, 'test_action')
+
+        mock_check_policy.assert_called_once_with(
+            self.req_context, 'test_action', {'project_id': 'test_project_id',
+                                              'user_id': 'user'})
```
Comment (Dany9966) [inline coriolis/tests/test_context.py:74]: Please also add a test case where you pass a custom `target`, and also `fatal=False`

**Review by Dany9966 — CHANGES_REQUESTED** [review]
Aside from some additions, LGTM

**Comment by mihaelabalutoiu** [comment]
> Also please add test for `get_admin_context`

Done,  I made the changes, as well. Thank you!

**Review by Dany9966 — APPROVED** [review]

**Pull request merged**